docs/source/enable_concepts.rst

```diff
@@ -90,18 +90,20 @@ Enable :class:`Component` and :class:`Container`.
 When a component gets an event, it tries to handle it in a standard way, which
 is to dispatch to:
 
- 1. its active tool
- 2. its overlays 
- 3. itself, so that any event handler methods on itself get called
- 4. its underlays
- 5. its listener tools
+1. its active tool
+2. its overlays
+3. itself, so that any event handler methods on itself get called
+4. its underlays
+5. its listener tools
 
 That logic is in :class:`Component`, in the :meth:`\_new_dispatch` method, which
 is called from :meth:`Component.dispatch` (:meth:`\_old_dispatch` will be
 removed in 3.0). If any of these handlers sets event.handled to True, event
 propagation stops. If an event gets as far as the listener tools, then all of
 them get the event.
 
+.. note::
+
   (The notion of an active tool is not used in current code, just older client

```

Comment:
Are the enclosing parentheses necessary now that this is rendered as a proper note?

Reply:
I was debating that myself.